spec/forms/find/university_degree_status_form_spec.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Find::UniversityDegreeStatusForm do
+  describe 'validations' do
+    it 'is valid when university_degree_status is present' do
+      form = described_class.new(university_degree_status: 'true')
+      expect(form).to be_valid
+    end
+
+    it 'is not valid when university_degree_status is not present' do
+      form = described_class.new(university_degree_status: nil)
+      expect(form).not_to be_valid
+      expect(form.errors[:university_degree_status]).to include(
+        'Select whether you have a university degree'
+      )
+    end
+  end
+end
```

Comment:
```suggestion
# frozen_string_literal: true

require 'rails_helper'

RSpec.describe Find::UniversityDegreeStatusForm, type: :model do
  describe 'validations' do
    it { is_expected.to validate_presence_of(:university_degree_status) }

    context 'when university_degree_status is not present' do
      subject { described_class.new(university_degree_status: nil) }

      it 'is not valid' do
        expect(subject).not_to be_valid
        expect(subject.errors[:university_degree_status]).to include(
          'Select whether you have a university degree'
        )
      end
    end
  end
end
```

Reply:
Huumm, I think this is the only comment that I don't agree with.

I will be honest, I never, never, never like shoulda-matchers. 

Although they could sound useful, I'd prefer to be explicitly with the behaviour on test. that's a personal preference.

I think this is the only comment I don't think I should address.

But happy to discuss and hear more about it.

Reply:
Ok, thats fine, i'm happy with your approach, it was only a suggestion. 

When there are lots of validations to be tested, I quite liked seeing them tested in one long concise list like this: 

https://github.com/DFE-Digital/publish-teacher-training/blob/623165b716afc60dc820d0f1b14ad26994f3e8ed/spec/models/course_spec.rb#L481-L491

We have used this approach quite a lot in Publish.